mslib/mscolab/models.py

```diff
@@ -183,6 +183,7 @@ def __init__(self, op_id, u_id, text, message_type=MessageType.TEXT, reply_id=No
         self.text = text
         self.message_type = message_type
         self.reply_id = reply_id
+        self.created_at = datetime.datetime.utcnow()
```

Comment:
Is this intentionally utcnow?  `datetime.datetime.now(tz=datetime.timezone.utc)` should behave the same in this case, I think.

Reply:
Also, since the created_at column has a default of `datetime.datetime.utcnow` I think this entire line is redundant.

Reply:
> Is this intentionally utcnow? `datetime.datetime.now(tz=datetime.timezone.utc)` should behave the same in this case, I think.

it is a bug in MSS-8.3.3 which has the old writings on all places. And I don't want to mix that now up. Because it gets refatored in develop.

Reply:
> Also, since the created_at column has a default of `datetime.datetime.utcnow` I think this entire line is redundant.

I thougt this too, but it only gets instanciated a new value, when I added the line. See the updated tests.

Reply:
> I thougt this too, but it only gets instanciated a new value, when I added the line.

That would be a bug in sqlalchemy then. But I cannot reproduce that behavior.

> See the updated tests.

The tests pass even without the changes to mslib/mscolab/models.py.